src/commonMain/kotlin/character/classes/boss/Boss.kt

```diff
@@ -14,7 +14,7 @@ class Boss(override var baseStats: Stats) : Class(mapOf(), BossSpec()) {
     }
 
     override var buffs: List<Buff> = listOf()
-    override var resourceType: Resource.Type = Resource.Type.MANA
+    override val resourceType: MutableList<Resource.Type> = mutableListOf(Resource.Type.MANA)
```

Comment:
Do these need to be `MutableList`s?  I'd prefer if they could be non-mutable `List` and constructed with `listOf()`

Reply:
All changed to List now, all the characters too.